app/assets/stylesheets/omni_nav/omni_nav_v2/off-canvas-nav.scss

```diff
@@ -93,6 +93,13 @@
             height: 23px;
             width: 75%;
           }
+
+          a.default.off-logo {
```

Comment:
This is the reason why umbrella categories that weren't school/colleges were missing the Chapman logo in the off canvas. They were using the class name "default off-logo" which didn't exist.